modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -789,9 +821,18 @@
 	cortical_owner.children_produced++
 	if(cortical_owner.children_produced == GLOB.objective_egg_egg_number)
 		GLOB.successful_egg_number += 1
-	if(prob(25))
-		cortical_owner.human_host.gain_trauma_type(BRAIN_TRAUMA_MILD, TRAUMA_RESILIENCE_BASIC)
-		to_chat(cortical_owner.human_host, span_warning("Your brain begins to hurt..."))
+	var/obj/item/organ/internal/brain/victim_brain = cortical_owner.human_host.getorganslot(ORGAN_SLOT_BRAIN)
+	if(victim_brain)
+		cortical_owner.human_host.adjustOrganLoss(ORGAN_SLOT_BRAIN, 25)
+		if(prob(33))
+			cortical_owner.human_host.gain_trauma_type(BRAIN_TRAUMA_MILD, TRAUMA_RESILIENCE_BASIC)
+		else if(prob(25))
+			cortical_owner.human_host.gain_trauma_type(BRAIN_TRAUMA_MILD, TRAUMA_RESILIENCE_SURGERY)
+		else if(prob(10))
+			cortical_owner.human_host.gain_trauma_type(BRAIN_TRAUMA_SEVERE, TRAUMA_RESILIENCE_SURGERY)
+		else if(prob(3))
```

Comment:
yandere simulator moment-- a switch from a rand(1,100) might be better, I have no clue.

Reply:
drank from the neurine chalice and changed it to a switch